main.go

```diff
@@ -128,6 +128,36 @@ func mainError() error {
 	daemonCommand.PersistentFlags().String(f.Service.Azure.HostCluster.Tenant.PartnerID, "", "Partner ID used for the Control Plane cluster.")
 	daemonCommand.PersistentFlags().String(f.Service.Azure.HostCluster.VirtualNetwork, "", "Host cluster virtual network name.")
 	daemonCommand.PersistentFlags().String(f.Service.Azure.HostCluster.VirtualNetworkGateway, "", "Host cluster virtual network gateway name.")
+
+	daemonCommand.PersistentFlags().String(f.Service.Cluster.BaseDomain, "", "Cluster base domain without k8s/g8s prefixes.")
```

Comment:
It's probably impossible to have a good default for all of them, but having sane defaults for as many parameters as possible will help both e2e tests and running the operator locally.